manpages/mkfs.fat.8.in

```diff
@@ -20,176 +20,174 @@
 size.
-With this option, the \fIBLOCK-COUNT\fR must be given, because otherwise the
+With this option, the \fIBLOCK-COUNT\fP must be given, because otherwise the
 intended size of the filesystem wouldn't be known.
 The file created is a sparse file, which actually only contains the meta-data
 areas (boot sector, FATs, and root directory).
 The data portions won't be stored on the disk, but the file nevertheless will
 have the correct size.
 The resulting file can be copied later to a floppy disk or other device, or
 mounted through a loop device.
-.IP "\fB\-D\fR \fIDRIVE-NUMBER\fR" 4
+.IP "\fB\-D\fP \fIDRIVE-NUMBER\fP" 4
 Specify the BIOS drive number to be stored in the FAT boot sector.
-This value is usually 0x80 for hard disks and 0x00 for floppy devices or
-partitions to be used for floppy emulation.
-.IP "\fB\-f\fR \fINUMBER-OF-FATS\fR" 4
+.IP "\fB\-f\fP \fINUMBER-OF-FATS\fP" 4
 Specify the number of file allocation tables in the filesystem.
 The default is 2.
-.IP "\fB\-F\fR \fIFAT-SIZE\fR" 4
+.IP "\fB\-F\fP \fIFAT-SIZE\fP" 4
 Specifies the type of file allocation tables used (12, 16 or 32 bit).
-If nothing is specified, \fBmkfs.fat\fR will automatically select between 12, 16
+If nothing is specified, \fBmkfs.fat\fP will automatically select between 12, 16
 and 32 bit, whatever fits better for the filesystem size.
-.IP "\fB\-h\fR \fINUMBER-OF-HIDDEN-SECTORS\fR" 4
+.IP "\fB\-h\fP \fINUMBER-OF-HIDDEN-SECTORS\fP" 4
 Select the number of hidden sectors in the volume.
 Apparently some digital cameras get indigestion if you feed them a CF card
 without such hidden sectors, this option allows you to satisfy them.
-.IP "\fB\-i\fR \fIVOLUME-ID\fR" 4
-Sets the volume ID of the newly created filesystem; \fIVOLUME-ID\fR is a 32-bit
+.IP "\fB\-i\fP \fIVOLUME-ID\fP" 4
+Sets the volume ID of the newly created filesystem; \fIVOLUME-ID\fP is a 32-bit
 hexadecimal number (for example, 2e24ec82).
 The default is a number which depends on the filesystem creation time.
-.IP "\fB\-I\fR" 4
+.IP "\fB\-I\fP" 4
 It is typical for fixed disk devices to be partitioned so, by default, you are
 not permitted to create a filesystem across the entire device.
-\fBmkfs.fat\fR will complain and tell you that it refuses to work.
+\fBmkfs.fat\fP will complain and tell you that it refuses to work.
 This is different when using MO disks.
 One doesn't always need partitions on MO disks.
 The filesystem can go directly to the whole disk.
-Under other OSes this is known as the 'superfloppy' format.
-This switch will force \fBmkfs.fat\fR to work properly.
-.IP "\fB\-l\fR \fIFILENAME\fR" 4
-Read the bad blocks list from \fIFILENAME\fR.
-.IP "\fB\-m\fR \fIMESSAGE-FILE\fR" 4
+Under other OSes this is known as the \fIsuperfloppy\fP format.
+This switch will force \fBmkfs.fat\fP to work properly.
+.IP "\fB\-l\fP \fIFILENAME\fP" 4
+Read the bad blocks list from \fIFILENAME\fP.
+.IP "\fB\-m\fP \fIMESSAGE-FILE\fP" 4
```

Comment:
For introducing a jargon expression, i would use

    as the \(lqsuperfloppy\(rq format.